packages/libraries.python2.vm/tools/chocolateyinstall.ps1

```diff
@@ -1,4 +1,4 @@
-$ErrorActionPreference = 'Continue'
+$ErrorActionPreference = 'Stop'
```

Comment:
I assume that was here for a reason? Should I exclude this and the Py3 library package from this check?

Reply:
I think this is because we install several python libraries with this package and we want to install as many as possible even if one fails (as they are independent). So, yes, exclude them from the check.

Reply:
baae48a